qa/L0_backend_python/decoupled/decoupled_test.py

```diff
@@ -64,11 +68,50 @@ def test_decoupled_execute_error(self):
         shape = [2, 2]
         number_of_requests = 2
         user_data = UserData()
-        with grpcclient.InferenceServerClient("localhost:8001") as triton_client:
-            triton_client.start_stream(callback=partial(callback, user_data))
+        with self._shm_leak_detector.Probe(debug_str=model_name) as shm_probe:
```

Comment:
This file looks like a lot of changes, but it's just adding probe context around the large block of each test and then indenting it.